addon/components/o-s-s/iconName.enum.ts

```diff
@@ -104,6 +104,7 @@ export enum IconNames {
   Truck = 'fa-truck',
   Twitch = 'fa-twitch',
   Twitter = 'fa-twitter',
```

Comment:
Is this one still useful ?

Reply:
Technically no, I'll keep it just to make sure. I'll take it off some other time.